docs/topics/jvm/java-to-kotlin-interop.md

```diff
@@ -639,3 +639,27 @@ MyInt output = ExampleKt.timesTwoBoxed(input);
 
 To apply this behavior to all inline value classes and the functions that use them within a module, compile it with the `-Xjvm-expose-boxed` option. 
 Compiling with this option has the same effect as if every declaration in the module has the `@JvmExposeBoxed` annotation.
+
+> The `@JvmExposeBoxed` annotation doesn't automatically generate boxed representations for inherited functions.
+> 
+> To generate the necessary representation for an inherited function, override it in the implementing or extending class:
+> 
+> ```kotlin
```

Comment:
In the example, can you add names that reflect a real-world example a bit more? Maybe like:
```
interface IdTransformer {
    fun transformId(rawId: UInt): UInt = rawId
}

// Doesn't generate a boxed representation for the transformId() function
@OptIn(ExperimentalStdlibApi::class)
@JvmExposeBoxed
class LightweightTransformer : IdTransformer

// Generates a boxed representation for the transformId() function
@OptIn(ExperimentalStdlibApi::class)
@JvmExposeBoxed
class DefaultTransformer : IdTransformer {
    override fun transformId(rawId: UInt): UInt = super.transformId(rawId)
}
```
And is there a way to avoid using `super` ? Because we don't explain it on this page 🤔